flixel/input/touch/FlxTouch.hx

```diff
@@ -38,30 +50,42 @@ class FlxTouch extends FlxPointer implements IFlxDestroyable implements IFlxInpu
 	{
 		input = null;
 		justPressedPosition = FlxDestroyUtil.put(justPressedPosition);
+		_pressure = null;
 		flashPoint = null;
 	}
 
 	/**
-	 * Resets the justPressed/justReleased flags and sets touch to not pressed.
+	 * Resets the justPressed/justReleased flags, sets touch to not pressed and sets touch pressure to 0.
 	 */
-	public function recycle(x:Int, y:Int, pointID:Int):Void
+	public function recycle(x:Int, y:Int, pointID:Int, pressure:Float):Void
 	{
 		setXY(x, y);
 		input.ID = pointID;
 		input.reset();
+		setPressure(pressure);
+		if (pressure > 0)
```

Comment:
remove the pen property and just set pressure to 1.0, in `FlxTouchManager` when pressure is 0.0